Index: ClassyCrafT/ClassyCrafT/src/main/java/raf/dsw/classycraft/app/AppCore.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package raf.dsw.classycraft.app;\r\n\r\nimport raf.dsw.classycraft.app.core.ApplicationFramework;\r\n\r\npublic class AppCore {\r\n    public static void main(String[] args) {\r\n        ApplicationFramework appCore = ApplicationFramework.getInstance();\r\n        appCore.initialize(); //\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ClassyCrafT/ClassyCrafT/src/main/java/raf/dsw/classycraft/app/AppCore.java b/ClassyCrafT/ClassyCrafT/src/main/java/raf/dsw/classycraft/app/AppCore.java
--- a/ClassyCrafT/ClassyCrafT/src/main/java/raf/dsw/classycraft/app/AppCore.java	(revision 27ad0d9ee85eee64babbb06c8f97760dec087c9b)
+++ b/ClassyCrafT/ClassyCrafT/src/main/java/raf/dsw/classycraft/app/AppCore.java	(date 1698269899703)
@@ -5,6 +5,6 @@
 public class AppCore {
     public static void main(String[] args) {
         ApplicationFramework appCore = ApplicationFramework.getInstance();
-        appCore.initialize(); //
+        appCore.initialize();
     }
 }
\ No newline at end of file
